### DISCUSSION TITLE
Categories not (immediately) showing for recurrent journal entries

### DISCUSSION THREAD

**Discussion by DickinsonSa** [TechbeeAT/jtxBoard#374] [General]
Categories are present in journal entries but only appear once...

1. Edit entry
2. Go to series
3. Select edit
4. Select save (bottom right icon)
5. Select top left arrow to go back
(dismiss SORRY message by selecting'back') - btw this message appears all too often! 
6. Category is now visible

If I knew how to create a log, I would. If there is a way, please tell me. 

About/licence: 2.03.00-rc22.gplay (203000022)
Release notes state version as 2.3.0
Android v12
Galaxy tab a7

Hope this is of some help.

**Comment by patrickunterwegs** [comment]
Hey @DickinsonSa ,

So do I get it right, you have set up a series, you go back to the series definition and edit it by adding another category. You save by going back but the change is only saved in one instance?
Have you created the series before or after the update to 2.3.0? 
May I ask you to double-check if you can see the same behavior with series that were created with 2.3.0?


About the sorry message: yes, indeed I need to do something about it. The reason is that on back you'd return to the previous entry. But if the previous entry was a recurring instance and the series was changed, all (unchanged) instances get deleted and recreated (with the change). So technically the previous entry really doesn't exist anymore. 
Anyhow, this is just for explanation, I'll check a solution for that.

**Reply by DickinsonSa** [reply]
Hi Patrick,
These are old recurrent entries created prior to the newer version.
I can try running a test for new entries tomorrow for you, but for now, they're all old.

**Reply by patrickunterwegs** [reply]
Hey @DickinsonSa , did you encounter this issue again?
